scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -443,3 +443,37 @@ <h6>Phenotype terms (HPO):</h6>
 	</tbody>
 </table>
 {% endmacro %}
+
+
+{% macro gene_panels(variant) %}
+<strong>Gene panels:</strong><br>
+{% for panel in variant.case_panels|sort(attribute='panel_name', case_sensitive=False)|rejectattr('removed') %}
+  {% set no_symbol_genes_counter = namespace(value=0) %}
+  <a
+    href="{{ url_for('panels.panel', panel_id=panel.panel_id) }}"
+    {#
+    Add hovertip listing occurence of each gene on selected panels, in the case
+    where variant covers multiple genes.
+    #}
+    {% if (variant.genes | length) > 1 %}
+    class="badge bg-secondary text-white"
+    data-bs-original-title="Genes on gene panel"
+    data-bs-toggle="popover"
+    data-bs-html="true"
+    data-bs-trigger="hover click"
+    data-bs-content="
+    {% for gene in variant.genes %}
+      {% if panel['panel_name'] in gene['associated_gene_panels'] %}
+        {% if gene.common %}
+          {{ gene.common.hgnc_symbol }}
+        {% else %}
+          {% set no_symbol_genes_counter.value = no_symbol_genes_counter.value + 1 %}
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+    {% if no_symbol_genes_counter.value > 0 %}&nbsp; + {{no_symbol_genes_counter.value}} other genes {% endif %}
```

Comment:
```suggestion
    {% if no_symbol_genes_counter.value > 0 %}&nbsp; + {{no_symbol_genes_counter.value}} other genes on this panel {% endif %}
```

Reply:
I still have to fix for when that list has no genes with symbol so you'll have something like this, which is ugly:

 "blank" + 456 other genes